toolchains/regenerate.sh

```diff
@@ -38,7 +38,7 @@ esac
 
 # Bazel query is the right command so bazel won't fail itself.
 # Keep bazel versions here at most two: current master version, next version
-for BAZEL_VERSION in "3.3.1" "3.4.1"; do
+for BAZEL_VERSION in "3.5.1" "3.6.0"; do
```

Comment:
Let's keep the first as `3.4.1` so it won't block us updating build-tools sha in envoyproxy/envoy.